planetary-social/nos
# Add new featured authors and new categories to the Discover tab

## Issues covered
#1592 
#1663

## Description
Adds new featured authors and new categories (in alphabetical order) to the `Discover` tab. Some authors were also removed from the list.

## How to test
1. Build the code.
2. Tap the Discover tab at the bottom of the screen.
3. Please check that there are new categories and featured authors on this screen. The categories and authors can be cross-checked with the list [here](https://docs.google.com/spreadsheets/d/1tyeqgborx7dpTbV-fQI7KAK0nXISgpQ_qfRt2edoqRg/edit?gid=0#gid=0). Authors in red are ignored for now.
4. Please check that the authors [here](https://docs.google.com/spreadsheets/d/1tyeqgborx7dpTbV-fQI7KAK0nXISgpQ_qfRt2edoqRg/edit?gid=1946470827#gid=1946470827) are no longer featured.

## Screenshots/Video
| Before | After |
|--------|-------|
| <img src="https://github.com/user-attachments/assets/6ccd69f6-00db-4c3f-850f-adc886114ea8" alt="Before" width="250"/> | <img src="https://github.com/user-attachments/assets/e011a261-d875-4dec-a978-4d9cf3e03bf6" alt="After" width="250"/> |
